### ISSUE TITLE
Lesson visibility filters appear to be inverted

### ISSUE THREAD

**Issue by rtibbles** [learningequality/kolibri#10483]
Tested in Firefox on Ubuntu 20.04, string seems to be correctly displayed now, but there's something really weird in how the lesson filters work, although I'm unsure this is a result of your PR 😕 

https://user-images.githubusercontent.com/1457929/232158839-def5fac1-1bcb-45fd-823d-88298c3406dc.mp4

---
Visible lesson is displayed as a result of filtering lessons to not visible 😳

_Originally posted by @radinamatic in https://github.com/learningequality/kolibri/issues/10476#issuecomment-1509301835_

**Comment by thanksameeelian** [comment]
> Tested in Firefox on Ubuntu 20.04, string seems to be correctly displayed now, but there's something really weird in how the lesson filters work, although I'm unsure this is a result of your PR 😕 


just confirming the continued presence of a bug surrounding visibility, as at times i've had trouble replicating any problems while trying to find/address the bug. 

video attached of the second user flow that caused an issue: mismatch of "visible"/"not visible" value selected in the dropdown vs. the lesson's visibility + inability to (at least visually) change lesson visibility when in "not visible" view:

https://user-images.githubusercontent.com/43046460/232591484-983a1dc8-df8f-4b70-8ae7-752677fd7e68.mov


the first time running through it, i encountered a scenario i haven't been able to reproduce again:
```
- if there is a lesson set to visible
    - can see in “visible” view ✅
    - can’t see in “not visible” view  ✅
    - —> set lesson to “not visible”
        - can’t see in “visible” view  ✅
        - can’t see in “not visible” view ❌
        --> after a period of time, lesson switches _itself_ to visible, changing the toggle, and now visibility is aligned with string selected in dropdown
```
then it behaved completely as expected through several hard page refreshes, navigation away and back, etc. only was able to encounter the second problem in the screen recording after wholly stopping and restarting my kolibri, though this may be a red herring.

@rtibbles interested to hear if there was a specific user interaction or other cases that alerted you to the weirdness happening here, though you ended up seeing the expected string. having slight issues consistently reproducing/testing out lil fixes

**Comment by rtibbles** [comment]
I am just the messenger - @radinamatic originally reported this!

**Comment by radinamatic** [comment]
@thanksameeelian Are you running from the latest develop? Just saying because I don't see the **Make lesson visible** alert modal when you toggle the visibility, but you may have checked the _Don't show this message again_ ...

Anyways, may I suggest you start debugging from the difference in how and why the lesson **Status** filter is working correctly on the **Plan** tab and is not working in the **Report tab**? In this latest recording it does not look inverted anymore, but is more like setting the filter to _Visible_ is filtering out everything and displaying _No results_, and setting the filter to _Not visible_ is not filtering anything and displaying lessons with all statuses... 😕 


https://user-images.githubusercontent.com/1457929/232606320-2d706ba8-f608-46f5-8d3f-0bc1826a3f8e.mp4

**Comment by thanksameeelian** [comment]
thanks @radinamatic!! i have the little message on the bottom activated vs the modal as a lesson's visibility switches. 

i discovered what i believe to be the root issue, which is that we are inconsistently trying to reference a property value - we're looking for `is_active` but the property we should be looking at is `active` in this case, and because the use of one or the other is slightly inconsistent the behavior can act pretty different depending on the steps one has taken. there's a place where we're relying on `!lesson.is_active` but since `lesson.is_active` is `undefined`, that is currently always resolving to `true` regardless of the actual state of the toggle, which impacts the behavior. 

a fix is incoming! :)

**Comment by thanksameeelian** [comment]
fixed in #10498
